ci_example_python/demos/demo_ros_pid.py

```diff
@@ -0,0 +1,56 @@
+#! /usr/bin/python
+
+##@package ci_example_python
+# 
+# @file demo_ros_pid.py
+# @author Vincent Berenz
+# @author Maximilien Naveau
+# license License BSD-3-Clause
+# @copyright Copyright (c) 2019, New York University and Max Planck Gesellschaft.
+# @date 2019-05-22
+#
+# @brief This demos present the API of the PID class using ROS param.
+#
+
+##@example demo_ros_pid.py
+#
+# In order to run this demos the `ci_example_python/python/ci_example_python` path
+# needs to be added in the `PYTHONPATH`. This is done after:
+# - 1. "catkin_make" is called from the root of the catkin workspace
+# - 2. "source ./devel/setup.bash" is called from the root of the catkin workspace
+# - 3. roscore is called in an additional terminal
+# - 4. Finally run `python ./demos_ros_pid.py`
+#
+# Notice the use of the double ## for the comments. This avoids doxygen
+# warnings, and allow you to explain in details the content of the demo.
+#
+
+import rospy
+from ci_example_python.pid import get_ros_params_pid
+
+
+if __name__ == "__main__":
+    
+    # here we set the parameters to ROS.
+    rospy.set_param('kp', 1.0)
+    rospy.set_param('kd', 1.0)
+    rospy.set_param('ki', 1.0)
+
+    ## Input position for the pid controller
+    current_position = 1
+    ## Input velcoity for the pid controller
+    current_velocity = 0.1
+    ## Input target position for the pid controller
+    position_target = 0
+    ## Input smapling period for the pid controller
+    delta_time = 0.001
+
+    print "pid using ros parameter server"
```

Comment:
I just noticed the Python 2 prints. Probably too much for this PR but given that Python 2 reaches EOL in a few weeks, we should at least use `from __future__ import print_function, division` here (this is anyway requested by the style guide :))
Best would actually to upgrade to Python 3 but this would probably be difficult with the use of `rospy`.

Reply:
Very good point! I updated that. It was not so hard^^